### ISSUE TITLE
No dropout on output for Regressor Models

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#303]
Reading through the fully-connected model code carefully, I just noted that the `TensorflowMultiTaskRegressor` classes don't put dropout after the output weights that feed into the L2 loss. On the other hand `TensorflowMultiTaskClassifier` does use dropout before feeding into cross-entropy loss. Need to run benchmark suite on regression with and without change to gauge effect of change.

@miaecle: Would it be possible to add command line arguments to the benchmarks to make it possible to run only regression datasets or only classification datasets? Could be useful as we try to pin down the behavior of our regression models carefully. Let's discuss more IRL this week.

**Comment by miaecle** [comment]
@rbharath Sure I can do that. Currently there are only classification datasets in the benchmark, which regression ones should we merge into it?

**Comment by rbharath** [comment]
@miaecle Delaney and Kaggle are already in `deepchem/examples`, so let's start with those. Would you mind adding those to the benchmark suite?

**Comment by joegomes** [comment]
@miaecle @rbharath It looks like the PR mentioned (#306) has been merged but there still isn't dropout on the output layer of `TensorflowMultiTaskRegressor`.  Is this still an issue?

**Comment by rbharath** [comment]
These models are now deprecated (in favor of `MultitaskRegressor`) so going to close.
